src/modules/ProductPage/ProductPage.tsx

```diff
@@ -0,0 +1,315 @@
+import { Breadcrumbs } from '../../components/Breadcrumbs';
+import { ProductsSlider } from '../../components/ProductsSlider';
+import { HeadingLevel } from '../../types/HeadingLevel';
+import { Title } from '../../components/Title';
+import { ProductSlider } from '../../components/ProductSlider';
+import { Product } from '../../types/Product';
+import { Link, useLocation, useParams } from 'react-router-dom';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import classNames from 'classnames';
+import { cartSlice } from '../../features/cart';
+import { favouriteSlice } from '../../features/favourite';
+import { BackButton } from '../../components/BackButton';
+import { useEffect, useState } from 'react';
+import {
+  getAccessories,
+  getAllProducts,
+  getPhones,
+  getTablets,
+} from '../../api/products';
+import { ProductData } from '../../types/ProductData';
+
+export const ProductPage = () => {
```

Comment:
![image](https://github.com/user-attachments/assets/153e0eb6-3dce-4076-8313-8839cd190014)

To wszystko pobiera ci sie na stronie produktu. Niektore rzeczy pobieraja sie dwa razy.